docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -48,6 +48,7 @@ To add additional clusters, increment the `0` value for each variable (`CAMUNDA_
 | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `BEARER_TOKEN`<br/><br/>(replaces [deprecated method `OAUTH`](/reference/announcements-release-notes/870/870-announcements.md#deprecated-web-modeler-cluster-authentication-oauth-and-client_credentials-self-managed)) | Web Modeler sends the authenticated user's token with every request to one of the cluster components (Zeebe, Operate, Tasklist).                                                        | The identity provider supports access tokens with multiple audiences.<br/>(_Note_: For the token to be accepted by the different cluster components, it must contain each component's audience.)<br/><br/>Example provider: Keycloak |
 | `CLIENT_CREDENTIALS`<br/><br/>([deprecated](/reference/announcements-release-notes/870/870-announcements.md#deprecated-web-modeler-cluster-authentication-oauth-and-client_credentials-self-managed))                   | Web Modeler requests an M2M token using the client credentials flow and sends this token with every request. The client ID and client secret have to be provided by the user in the UI. | The identity provider does not support access tokens with multiple audiences.<br/><br/>Example provider: [Microsoft Entra ID](/self-managed/setup/guides/connect-to-an-oidc-provider.md?authPlatform=microsoftEntraId#configuration) |
+| `BASIC`                                                                                                                                                                                                                 | Web Modeler sends a username and password with every request to one of the cluster components (Zeebe, Operate, Tasklist).                                                               | The identity provider supports basic authentication.                                                                                                                                                                                 |
```

Comment:
I don't think an (external) identity provider is used if basic authentication is enabled. So I would probably rephrase the "When to use?" part:

> The cluster uses basic authentication.